packages/redux/src/promotionEvaluations/reducer.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @module promotionEvaluations/reducer
+ * @category Promotion Evaluations
+ * @subcategory Reducer
+ */
+import * as actionTypes from './actionTypes';
+import { AnyAction, combineReducers } from 'redux';
+import type {
+  FetchPromotionEvaluationItemsFailureAction,
+  FetchPromotionEvaluationItemsRequestAction,
+  FetchPromotionEvaluationItemsSuccessAction,
+  State,
+} from './types';
+
+export const INITIAL_STATE: State = {
+  id: null,
+  error: null,
+  isLoading: false,
+  result: null,
+};
+
+const error = (
+  state = INITIAL_STATE.error,
+  action:
+    | FetchPromotionEvaluationItemsRequestAction
+    | FetchPromotionEvaluationItemsFailureAction
+    | FetchPromotionEvaluationItemsSuccessAction,
```

Comment:
In situations like this you probably just need to use `FetchPromotionEvaluationItemsAction`

Reply:
Done 🙂